src/modules/app/fetchData.js

```diff
@@ -45,10 +45,10 @@ async function fetchData<P, T> (
     return payload
   } catch (e) {
     let error
-    if (e instanceof LoadingError || e instanceof ParamMissingError || e instanceof MappingError) {
+    if (e instanceof ResponseError || e instanceof ParamMissingError || e instanceof MappingError) {
```

Comment:
Did we rename this error?

Reply:
According to this commit it was renamed to ResponseError